### DISCUSSION TITLE
failed to import gdal

### DISCUSSION THREAD

**Discussion by SolSeyoum** [giswqs/geospatial#52] [Q&A]
I followed to install geospatial package following the instructions however when I tried to import gdal, it failed with the following error.
from osgeo import gdal
error:  AttributeError: module 'osgeo._ogr' has no attribute 'OLCFastGetExtent3D'

I tried to downgrade the gdal package but it conflicts with several packages. Is there any solution for this?

Best regards,

**Comment by giswqs** [comment]
How did you install the package?
